BUILD

```diff
@@ -72,29 +73,11 @@ test_suite(
     tests = [
         "//base:unit_tests",
         "//clwb:unit_tests",
-        "//clwb:integration_tests",
+        "//clwb:headless_tests",
         "//cpp:unit_tests",
         "//dart:unit_tests",
         "//python:unit_tests",
         "//skylark:unit_tests",
     ],
 )
 # END-EXTERNAL
-
-# ASwB tests, run with an Android Studio plugin SDK
```

Comment:
Why?

Reply:
Because we can't even build aswb anymore. No use to keep tests around in my opinion.